packages/api/src/api/contract/contract.controller.spec.ts

```diff
@@ -278,7 +278,7 @@ describe("ContractController", () => {
         contractname: "contracts/HelloWorld.sol:HelloWorld",
         compilerversion: "0.8.17",
         optimizationUsed: "1",
-        zkCompilerVersion: "v1.3.14",
+        zksolcVersion: "v1.3.14",
```

Comment:
Does this mean that the change is breaking? If so, can we make it backward-compatible (e.g. add support for property alias)?

Reply:
Yes, this change is breaking. TBH I would better leave `zkCompilerVersion` for consistency and ask etherscan to rename on their side, because for solc it's not `solcVersion`, it's `compilerversion`. If we still decide to go with `zksolcVersion` then we should support both `zksolcVersion` and `zkCompilerVersion` to avoid breaking changes.